Pages/Index.cshtml.cs

```diff
@@ -27,73 +27,69 @@ public string GetData(string endpoint)
 
         public void OnGet()
         {
-            using (var WebClient = new WebClient())
+
+            string jsonstring = GetData("https://data.cityofchicago.org/resource/xq83-jr8c.json");
+            //               string jsonstring = WebClient.DownloadString("https://data.cityofchicago.org/resource/xq83-jr8c.json");
+            Energy[] energies = Energy.FromJson(jsonstring);
+
+            jsonstring = GetData("https://data.cityofchicago.org/resource/tfm3-3j95.json");
+            //                jsonstring = WebClient.DownloadString("https://data.cityofchicago.org/resource/tfm3-3j95.json");
+            Vehicle[] vehicles = Vehicle.FromJson(jsonstring);
+
+            List<Affluence> affluence = new List<Affluence>();
+
+            var energy_query = from energy in energies
+                               group energy by new { energy.ZipCode, energy.Latitude, energy.Longitude } into g
+                               select new
+                               {
+                                   Zip = g.Key.ZipCode,
+                                   Latitude = Math.Round(g.Key.Latitude, 0),
+                                   Longitude = Math.Round(g.Key.Longitude, 0),
+                                   PowerUsage = Math.Round(g.Sum(u => u.ElectricityUseKbtu) / g.Sum(u => u.GrossFloorAreaBuildingsSqFt), 0)
+
+                                   ////FloorSize = energy.GrossFloorAreaBuildingsSqFt,
+                                   //TotalUsage = g.Sum(u=> u.ElectricityUseKbtu),
```

Comment:
Remove commented out code.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.